lib/modules/platform/scm-manager/index.spec.ts

```diff
@@ -0,0 +1,650 @@
+import * as httpMock from '../../../../test/http-mock';
+import { git, mocked } from '../../../../test/util';
+import * as hostRules from '../../../util/host-rules';
+import type { Pr } from '../types';
+import * as _util from '../util';
+import { mapPrFromScmToRenovate } from './mapper';
+import type { PrFilterByState, PullRequest, Repo, User } from './types';
+import {
+  addAssignees,
+  addReviewers,
+  createPr,
+  deleteLabel,
+  ensureCommentRemoval,
+  ensureIssue,
+  ensureIssueClosing,
+  findIssue,
+  findPr,
+  getBranchPr,
+  getBranchStatus,
+  getBranchStatusCheck,
+  getIssueList,
+  getJsonFile,
+  getPr,
+  getPrList,
+  getRawFile,
+  getRepoForceRebase,
+  getRepos,
+  initPlatform,
+  initRepo,
+  invalidatePrCache,
+  massageMarkdown,
+  maxBodyLength,
+  mergePr,
+  setBranchStatus,
+  updatePr,
+} from './index';
+
+jest.mock('../../../util/git');
+jest.mock('../util');
+const util: jest.Mocked<typeof _util> = mocked(_util);
```

Comment:
```suggestion
const util = mocked(_util);
```

should not be required